apps/bc-wallet-showcase-creator/components/credentials/credentials-display.tsx

```diff
@@ -70,7 +72,7 @@ export const CredentialsDisplay = ({ searchTerm }: CredentialsDisplayProps) => {
               {openId === item.id ? (
                 <div className="p-3 bg-foreground/10 flex flex-col dark:bg-dark-bg items-center text-center transition-all duration-300">
                   <Image
-                    src={item.icon?.id?.trim() ? `${baseUrl}/assets/${item.icon.id}/file` : '/assets/no-image.jpg'}
+                    src={item.icon?.id?.trim() ? `${baseUrl}/${tenantId}/assets/${item.icon.id}/file` : '/assets/no-image.jpg'}
```

Comment:
is not easier to add `tenantId` to the `baseUrl` instead of to all the places that is been implemented?